proto/cosmos/accounts/v1/tx.proto

```diff
@@ -55,3 +60,21 @@ message MsgExecuteResponse {
   // response is the response returned by the account implementation.
   bytes response = 1;
 }
+
+// -------- Account Abstraction ---------
```

Comment:
we should be careful to not conflate bundler and AA nomenclature, it could cause confusion later on